### ISSUE TITLE
Wolfram|Alpha answer is difficult to understand

### ISSUE THREAD

**Issue by NeoMahler** [CatIRCBots/Granota#29]
> [20:54:21] -NeoMahler- .wa 2+2
> [20:54:22] -Granota- [WOLFRAM] `<html><head><meta http-equiv="content-type" content="text/html = charset=utf-8"><title>404 Not Found</title></head><body text=#000000 bgcolor=#ffffff><h1>Error: Not Found</h1><h2>The requested URL <code>/wa/2%252B2</code> was not found on this server.</h2><h2></h2></body></html>`

**Comment by NeoMahler** [comment]
The bot didn't connect directly to wolfram alpha, it used to connect to another age (http://tumbolia.appspot.com/wa/) that's doesn't exist now. In the wolfram alpha web page, I see that I need an API key to use the API. That's no problem for me, but that means that everybody will have to register a new API key.

The same error occurs in the .py function, because the bot connected to http://tumbolia.appspot.com/py/

**Comment by ElGatoSaez** [comment]
You can set blank parameters to the code if people want the WA module. You
can set your API Key for Granota in #catbots@irc.lizardirc.org

And if we need a calculator, we will search for one with an open API.

2015-09-29 6:16 GMT-03:00 NeoMahler notifications@github.com:

> The bot didn't connect directly to wolfram alpha, it used to connect to
> another age (http://tumbolia.appspot.com/wa/) that's doesn't exist now.
> In the wolfram alpha web page, I see that I need an API key to use the API.
> That's no problem for me, but that means that everybody will have to
> register a new API key.
> 
> The same error occurs in the .py function, because the bot connected to
> http://tumbolia.appspot.com/py/
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CatIRCBots/Granota/issues/29#issuecomment-144002203.

## 

_Saludos Coordiales_
_Sebastián Sáez M._

**Comment by NeoMahler** [comment]
Yes, I will probably set an API key for Granota... but that will be bad for newbies.

About the calculator, I don't see any problem at the moment, because it seems that Google calc has an open API. It works well with simple operations.

**Comment by jeda** [comment]
Probably we can run a service for using Wolfram without API key, and configure it on Granota.

**Comment by NeoMahler** [comment]
The problem is that out VPS is quite unstable... but maybe that's the only option if we don't want an API key if we don't find another open API.

**Comment by jeda** [comment]
It isn't.

14:29:02 up 12 days, 17:22,  0 users,  load average: 0.00, 0.01, 0.05

Also remember I have my Linode.

**Comment by NeoMahler** [comment]
Ah, ok with the Linode one. But I don't trust very much in the ovh vps... if you want to start coding that service, you're welcome to do it :smiley_cat:

**Comment by jeda** [comment]
Don't worry about the VPSes.

well, I dude about the service, lol

**Comment by NeoMahler** [comment]
@Worldev/developers I'm trying to fix it with the wolframalpha library. When I run it I get `ImportError: No module named jaraco.itertools (file "/usr/local/lib/python2.7/dist-packages/wolframalpha/__init__.py", line 7, in <module>)`, but jaraco.itertools is already installed. 

Any ideas?

**Comment by NeoMahler** [comment]
@Worldev/developers I'm trying to fix it with the wolframalpha library. When I run it I get `ImportError: No module named jaraco.itertools (file "/usr/local/lib/python2.7/dist-packages/wolframalpha/__init__.py", line 7, in <module>)`, but jaraco.itertools is already installed.   

Any ideas?

**Comment by NeoMahler** [comment]
The bug has been fixed but it needs enhancement, so I'm changing the label and the title but not closing the issue.

**Comment by NeoMahler** [comment]
The current output of the .wa command is:

> -NeoMahler- .wa 2+2
> -Granota- [WOLFRAM] 2+2 ▐ 4 ▐ four ▐ | + |  |  =  | 2 |  | 2 |  | 4 ▐ age 6:  3.2 seconds  |  age 8:  1.8 seconds  |  age 10:  1.2 seconds  |  age 18:  0.83 seconds(ignoring concentration, repetition, variations in education, etc.)

The "| + |  |  =  | 2 |  | 2 |  | 4", for example, is quite impossible to understand (yes, it's 2+2=4, but still...). Same with the next section...
